provider_test.go

```diff
@@ -39,4 +39,8 @@ func testAccPreCheck(t *testing.T) {
 	if v := os.Getenv("TF_VAR_domain"); v == "" {
 		t.Fatal("TF_VAR_domain must be set for acceptance tests")
 	}
+
+	if v := os.Getenv("TF_VAR_private_network_name"); v == "" {
+		t.Fatal("TF_VAR_private_network_name must be set for acceptance tests")
```

Comment:
I would rather that we skip tests that require this variable, instead of disabling all tests if the variable is not provided. As not everyone (like me) will have a private netwerk configured. See earlier comment on how to enable per test skips.

Reply:
I've changed `t.Fatal` to `t.Skip` here as well, would that suffice?